mlir/lib/Dialect/MQTOpt/Transforms/QuantumSinkPass.cpp

```diff
@@ -37,7 +36,7 @@ struct QuantumSinkPass final : impl::QuantumSinkPassBase<QuantumSinkPass> {
     populateQuantumSinkPushPatterns(patterns);
 
     // Apply patterns in an iterative and greedy manner.
-    if (mlir::failed(mlir::applyPatternsGreedily(op, std::move(patterns)))) {
+    if (mlir::failed(APPLY_PATTERNS_GREEDILY(op, std::move(patterns)))) {
```

Comment:
The `#include <mlir/Support/LLVM.h>` include was erroneously removed from this file.